[11042022 15:52:26] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 15:52:26] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 15:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 15:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 15:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 15:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 15:57:22] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 15:57:25] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 15:57:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 15:58:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 15:59:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:00:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:01:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:02:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:03:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:04:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:05:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:06:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:07:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:07:59] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:08:02] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:08:02] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:09:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:09:25] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:09:28] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:09:28] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:10:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:11:01] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:11:04] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:11:04] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:12:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:13:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:14:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:15:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:16:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:17:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:18:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:18:47] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:18:50] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:18:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:19:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:20:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:21:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:22:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:23:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:24:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:25:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:26:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:27:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:28:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:29:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:30:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:31:49] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[11042022 16:31:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:32:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:33:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:34:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:35:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:36:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:37:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:38:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:39:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:40:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:41:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:42:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:43:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:44:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:45:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:46:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:47:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:48:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:49:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:50:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:51:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 16:52:32] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:52:37] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:52:37] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:52:39] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 16:53:28] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 16:53:28] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 16:54:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 17:00:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 17:00:49] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 17:00:49] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 17:00:49] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[11042022 18:41:30] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[11042022 18:41:30] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'EHLO''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b"\x16\x03\x01\x00\xf4\x01\x00\x00\xf0\x03\x03\x94@(\x1c\x9eMf\x94\xf07\xf4\x92F\\\xc3*h\x98\xf0\xff\xbc\xdb\xc8z\xaf\xbc\xec\x9aL'a\x8e"'
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'GET''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x16\x03\x01\x00\xf4\x01\x00\x00\xf0\x03\x03\xd2\x1f5\xc9\xb9\xb3\xc6\tU\xaf\xbdf\xb8\xbeC\x9bA\xbdG\x9d\xbf\xd2\xca|g\xf3\x87Q1\xf8\xdaU''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'GET''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Host:''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Accept:''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'User-Agent:''
[11042022 18:41:30] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Connection:''
[11042022 18:41:30] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 18:41:35] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 18:41:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 18:43:48] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:45:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:45:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:48:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:49:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:50:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:52:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:53:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:54:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:54:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:54:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 18:54:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 19:06:22] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 19:06:22] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[11042022 19:09:18] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[11042022 19:09:18] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[11042022 19:12:13] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[11042022 19:12:13] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
